contracts/oracles/UniswapV3PriceFeed.sol

```diff
@@ -0,0 +1,61 @@
+// SPDX-License-Identifier: MIT
+
+pragma solidity ^0.8.0;
+
+import "@chainlink/contracts/src/v0.8/interfaces/FeedRegistryInterface.sol";
+import "@chainlink/contracts/src/v0.8/Denominations.sol";
+import "../interfaces/IMultipriceOracle.sol";
+import "../interfaces/IPriceFeed.sol";
+
+contract UniswapV3PriceFeed is IPriceFeed {
+    address public registry;
+    address public multiPriceOracle;
+    address public token;
+
+    address public constant weth = 0xA0b86991c6218b36c1d19D4a2e9Eb0cE3606eB48;
+    uint public constant amountIn = 1e18;
+    uint32 public constant twapPeriod = 86400; // 1 day
```

Comment:
how about
```suggestion
    uint32 public constant twapPeriod = 1 days;
```
https://docs.soliditylang.org/en/v0.8.6/units-and-global-variables.html#time-units